### ISSUE TITLE
Append libraries root variable to CMAKE_PREFIX_PATH

### ISSUE THREAD

**Issue by osechet** [conan-io/conan#291]
CMAKE_PREFIX_PATH is a standard way to tell cmake where libraries, tools, etc, are located. Just append the root path of your library and the find_xxx() functions will look inside.

I'm trying to integrate conan in an existing project based on cmake. The dependencies are organized in a way similar to what conan do: a directory per library.
When calling cmake, we initialize the CMAKE_PREFIX_PATH variable with each library path, i.e:
CMAKE_PREFIX_PATH=/path/to/boost:/path/to/zlib
Then in the CMakeLists.txt, when needed, we call find_package() and CMake finds the include path, the libs, the tools for us.
To integrate conan, I changed the main CMakeLists.txt file of our project to update the variable after calling CONAN_BASIC_SETUP(). However, I will have to update it each time a library will be added or removed.
```
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
CONAN_BASIC_SETUP()

list(APPEND CMAKE_PREFIX_PATH ${CONAN_PROTOBUF_ROOT})
```

I think conan should append the root path of each library to CMAKE_PREFIX_PATH in the conanbuildinfo.cmake file.

**Comment by memsharded** [comment]
Yes, agree, this has also (co-laterally) been requested in an open issue: https://github.com/conan-io/conan/issues/221

Let's try to make it into next release 0.10 (self-assigned, can't promise, but I'll try). Thanks very much!

**Comment by lasote** [comment]
Fixed in 0.10.0, please try it and reopen if needed.
